Riot/Modules/Onboarding/OnboardingCoordinator.swift

```diff
@@ -61,6 +61,13 @@ final class OnboardingCoordinator: NSObject, OnboardingCoordinatorProtocol {
     // MARK: Screen results
     private var splashScreenResult: OnboardingSplashScreenViewModelResult?
     private var useCaseResult: OnboardingUseCaseViewModelResult?
+    private var authenticationType: MXKAuthenticationType?
+    private var session: MXSession?
+    
+    /// Whether all of the onboarding steps have been completed or not. `false` if there are more screens to be shown.
+    private var onboardingIsFinished = false
+    /// Whether the main app is ready to be shown or not. `true` once authenticated, verified and the store/data sources are ready.
+    private var appIsReady = false
```

Comment:
Maybe we should reverse the meaning here and call it what it is: "authenticationFinished". On the same note I think `onboardingFinished` slightly better than `onboardingIsFinished`. If you really want a `is` in there is should go in the front e.g. `isOnboardingFinished` a la ObjC